src/gui/curses/headless/main.c

```diff
@@ -49,23 +49,20 @@ daemonize (void)
     pid_t pid;
     int fd, i;
 
-    printf (_("Running WeeChat in background..."));
-    printf (" ");
+    printf ("%s ", _("Running WeeChat in background..."));
```

Comment:
Hello,

I agree with these changes, but just curious why the compiler reports only these ones?
There are many other printf in the code where a translated string is given as argument, without a format string.

By the way `-Wformat -Werror=format-security` is always forced during build, but neither gcc nor clang give me the warning.

Also this could be an issue only if the translated string uses format chars, right?

Reply:
I did a `git grep 'printf.*(_'` and it seems like the only other translated strings are passed to `log_printf` rather than the libc `printf`, so the compiler is ignoring them. `gettext` is annotated on some libintl implementations with ` __attribute__ ((format_arg))` so that the compiler verifies the format string passed to `gettext` matches the remaining `printf` args, instead of throwing the warning. It's possible for a malicious translation to alter the format string, so locale data should be considered security-sensitive.